### ISSUE TITLE
[bug?] multiple nodes are nested inside each other

### ISSUE THREAD

**Issue by DanielDecker** [Qu4tro/homie-spec#1]
When creating a device with multiple nodes (2+) the output of device.messages() will report the node path of node2 inside of node1. eg.
```python
desktop = Device(id="desktop", name="Desktop Computer", nodes={"local-time": localtime, "hearthbeat" : hearthbeat, "temperatur" : temp})

for msg in desktop.messages():
    print(msg.attrs)
```

```
{'topic': 'homie/desktop/$nodes', 'payload': 'local-time,hearthbeat,temperatur', 'retained': True, 'qos': 1}
{'topic': 'homie/desktop/local-time/$name', 'payload': 'Local time', 'retained': True, 'qos': 1}
...
{'topic': 'homie/desktop/local-time/hearthbeat/$name', 'payload': 'Device hearthbeat', 'retained': True, 'qos': 1}
...
{'topic': 'homie/desktop/local-time/hearthbeat/temperatur/$name', 'payload': 'Temperatur', 'retained': True, 'qos': 1}
```
I assumed that for the above device all the nodes would reside directly below `homie/desktop/`. Am I wrong?

I can prepare a PR for this issue.

**Comment by Qu4tro** [comment]
No, you're absolutely right.

I never gotten to actually use this library, so I would greatly appreciate the PR.

Thank you for reporting the issue!

**Comment by DanielDecker** [comment]
was fixed with PR #2
